### ISSUE TITLE
Border settings

### ISSUE THREAD

**Issue by Famiu** [akiyosi/goneovim#150]
We should have a way of changing the color, style and thickness of the GUI border. I find it necessary because I'm not really a big fan of the current border style.

**Comment by akiyosi** [comment]
I implemented this feature in v0.4.7 branch.
If you are a MacOS or Linux user, you can check out the feature in the following branches I'd appreciate your feedback.

https://github.com/akiyosi/goneovim/actions/runs/220881644

To enable the feature, add the following settings to `~/.goneovim/setting.toml`

``` toml
[Editor]
...
DrawWindowSeparator = true
WindowSeparatorTheme = "dark"
# # If you comment out this, the color theme of the window separator will be ignored and the specified color will take effect
# WindowSeparatorColor = "#2222ff"
WindowSeparatorGradient = true
...
```

**Comment by akiyosi** [comment]
Merged.
